secretstores/alicloud/parameterstore/parameterstore.go

```diff
@@ -0,0 +1,178 @@
+// ------------------------------------------------------------
```

Comment:
Please use apache license, refer to other files.

Reply:
Updated